src/edu/ucsb/cs56/projects/games/fish_animation/Shark.java

```diff
@@ -45,7 +46,11 @@ public Shark(double x, double y) {
     */
     @Override 
     public double getXPos() {
-	return xPos;
+      if (xPos < 0)
```

Comment:
Your latest commit added a bunch of changes to this file.  I'm pretty sure they aren't part of this feature?

Reply:
Already deleted